util/strings_test.go

```diff
@@ -38,3 +38,10 @@ func TestCoalesce(t *testing.T) {
 	assert.Equal("one", CoalesceStrings(nil, "", "one", "two"))
 	assert.Equal("", CoalesceStrings(nil, "", ""))
 }
+
+func TestStringContainsSliceRegex(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
Adding assert to the top of a function is actually an outdated practice; can you just pass in t for each of the asserts below?